src/stories/ActionList.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import {
+  ServerIcon,
+  PlusCircleIcon,
+  TypographyIcon,
+  VersionsIcon,
+  SearchIcon,
+  NoteIcon,
+  ProjectIcon,
+  FilterIcon,
+  GearIcon
+} from '@primer/octicons-react'
+import {Meta} from '@storybook/react'
+import React from 'react'
+import styled from 'styled-components'
+import {theme, ThemeProvider} from '..'
+import {ActionList as _ActionList} from '../ActionList'
+import {Header} from '../ActionList/Header'
+import BaseStyles from '../BaseStyles'
+
+const ActionList = Object.assign(_ActionList, {
+  Header
+})
+
+const meta: Meta = {
+  title: 'Composite components/ActionList',
+  component: ActionList,
+  decorators: [
+    (Story: React.ComponentType): JSX.Element => (
+      <ThemeProvider theme={theme}>
```

Comment:
ThemeProvider uses the Primer theme by default

```suggestion
      <ThemeProvider>
```